src/runtime/internal/dev.ts

```diff
@@ -256,11 +256,48 @@ export class SvelteComponentTyped<
 	 */
 	$$slot_def: Slots;
 
-	constructor(options: IComponentOptions<Props>) {
+	constructor(options: ComponentConstructorParams<Props>) {
 		super(options);
 	}
 }
 
+/**
+ * Convenience type to get the type of a Svelte component. Useful for example in combination with
+ * dynamic components and `<svelte:element>`.
+ * 
+ * Example:
+ * ```html
+ * <script lang="ts">
+ * 	import { ComponentType, SvelteComponentTyped } from 'svelte';
```

Comment:
should be
```suggestion
 * 	import type { ComponentType, SvelteComponentTyped } from 'svelte';
 ```
 right?
 Newer versions of Svelte projects will throw a TypeScript error when types are importet without the `type` keyword.